### PR TITLE
ISPN-14663 HGET

### PR DESCRIPTION
**Pull request by jabolina**
https://issues.redhat.com/browse/ISPN-14663

### CODE REVIEW

File: server/resp/src/main/java/org/infinispan/server/resp/commands/hash/HGET.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+import io.netty.channel.ChannelHandlerContext;
+
+/**
+ * `<code>HGET key field</code>` command.
+ * </p>
+ * Find the value associated with field in the hash stored at key.
+ *
+ * @since 15.0
+ * @see <a href="https://redis.io/commands/hget/">Redis Documentation</a>
```
Comment (karesti) [inline server/resp/src/main/java/org/infinispan/server/resp/commands/hash/HGET.java:22]: I think we should be consistent in thejava doc. The @see seems better to me. I will refactor other commands in other PR

File: server/resp/src/main/java/org/infinispan/server/resp/commands/hash/HGET.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ */
+public class HGET extends RespCommand implements Resp3Command {
+
+   public HGET() {
+      super(3, 1, 1, 1);
+   }
+
+   @Override
+   public CompletionStage<RespRequestHandler> perform(Resp3Handler handler, ChannelHandlerContext ctx, List<byte[]> arguments) {
+      if (arguments.size() != 2) {
```
Comment (karesti) [inline server/resp/src/main/java/org/infinispan/server/resp/commands/hash/HGET.java:33]: I did the arity check PR. Please merge that before this one

File: server/resp/src/test/java/org/infinispan/server/resp/HashOperationsTest.java [hunk 3]
```diff
@@ -39,7 +41,9 @@ public void testHSET() {
       // Mixing update and new keys.
       assertThat(redis.hset("HSET", Map.of("key2", "other-value2", "key4", "value4"))).isEqualTo(1);
 
-      // TODO: check when get method added.
+      assertThat(redis.hget("HSET", "key1")).isEqualTo("other-value1");
+      assertThat(redis.hget("HSET", "unknown")).isNull();
+      assertThat(redis.hget("UNKNOWN", "unknown")).isNull();
 
       assertThat(redis.set("plain", "string")).isEqualTo("OK");
       assertThatThrownBy(() -> redis.hmset("plain", Map.of("k1", "v1")))
```
Comment (karesti) [inline server/resp/src/test/java/org/infinispan/server/resp/HashOperationsTest.java:49]: are we checking the good error here ?

**Review by karesti — CHANGES_REQUESTED** [review]

File: server/resp/src/main/java/org/infinispan/server/resp/commands/hash/HGET.java (hunk 2 above)
Comment (jabolina) [reply server/resp/src/main/java/org/infinispan/server/resp/commands/hash/HGET.java:33]: Taking a look.

**Comment by jabolina** [comment]
@karesti Updated.

**Review by karesti — APPROVED** [review]

**Pull request merged**